ROADMAP.md

```diff
@@ -0,0 +1,72 @@
+# Learning roadmap
+
+## Basic Erlang
+1. Read [Learn you some Erlang for great good](http://learnyousomeerlang.com/)
+2. Work on some [erlings](https://github.com/lambdaclass/erlings)
+3. https://medium.com/@jlouis666/erlang-and-code-style-b5936dceb5e4
+4. Read our [guidelines](https://github.com/lambdaclass/guidelines)
+4. Work on a medium size open source project
+
+## Advanced Erlang
+1. https://medium.com/@jlouis666/how-to-build-stable-systems-6fe9dcf32fc4
+2. https://medium.com/@jlouis666/stacking-theory-for-systems-design-2450e6300689
+3. http://prog21.dadgum.com/70.html
+4. https://medium.com/@jlouis666/erlang-string-handling-7588daad8f05
+5. http://jlouisramblings.blogspot.com.ar/2013/01/how-erlang-does-scheduling.html
+1. https://joearms.github.io/published/2013-04-02-Red-and-Green-Callbacks.html
+1. https://ferd.ca/rtb-where-erlang-blooms.html
+1. http://jlouisramblings.blogspot.com.ar/2010/11/on-erlang-state-and-crashes.html
+1. https://ferd.ca/it-s-about-the-guarantees.html
+1. http://steve.vinoski.net/blog/2011/03/23/dont-lose-your-ets-tables/
+1. http://armstrongonsoftware.blogspot.com.ar/2008/05/road-we-didnt-go-down.html
+1. https://hamidreza-s.github.io/erlang%20garbage%20collection%20memory%20layout%20soft%20realtime/2015/08/24/erlang-garbage-collection-details-and-why-it-matters.html
+1. http://notdennisbyrne.blogspot.com.ar/2008/04/sequence-and-order-in-erlang.html
+1. https://medium.com/@jlouis666/the-erlang-shell-ab8d8bec3972
+1. https://ferd.ca/queues-don-t-fix-overload.html
+1. https://www.erlang-in-anger.com/
+1. Whatever you can find in [spawned shelter](http://spawnedshelter.com/)
+
+
+## Elixir
+1. https://elixir-lang.org/crash-course.html
+1. https://elixir-lang.org/getting-started/introduction.html
+1. https://hexdocs.pm/phoenix/overview.html
+1. https://hexdocs.pm/phoenix/up_and_running.html#content
+1. https://hexdocs.pm/gen_stage/GenStage.html
+1. https://elixirschool.com/en/lessons/advanced/gen-stage/
+
+## Distributed systems and databases
+1. http://book.mixu.net/distsys/single-page.html
+1. Designing Data-intensive applications book
+1. https://bravenewgeek.com/from-the-ground-up-reasoning-about-distributed-systems-in-the-real-world/
+1. https://aphyr.com/posts/281-jepsen-on-the-perils-of-network-partitions
+1. MapReduce: Simplified Data Processing on Large Clusters 
+1. Dynamo: Amazon’s Highly Available Key-value Store
+1. Bigtable: A Distributed Storage System for Structured Data
+1. Time, Clocks and Ordering of Events in a Distributed System
+1. Unreliable failure detectors and reliable distributed systems
+
+### Cassandra
```

Comment:
@facundoolano what do you think about removing the cassandra links? we might added them to a more general repo related to sysdist links we recommend reading

Reply:
the same applies to kafka. what do you think?

Reply:
I agree, they're just here so we don't lose the curated list, but they belong somewhere else (not sure how that repo would look)

Reply:
Cool. Please @AminArria remove thoses and add this list to the general readme